src/Microsoft.DotNet.Build.CloudTestTasks/PackageFiles/CloudTest.targets

```diff
@@ -155,12 +155,12 @@
     </ItemGroup>
 
     <PropertyGroup>
-        <_DesktopParameter Condition="'$(TargetGroup)' == 'net46'">--xunit-test-type=desktop</_DesktopParameter>
+        <OtherRunnerScriptArgs Condition="'$(TargetGroup)' == 'net46'">$(OtherRunnerScriptArgs) --xunit-test-type=desktop</OtherRunnerScriptArgs>
```

Comment:
LGTM other than to be defensive against others who use the property wrong you might include one more final space.